src/Symfony/Bridge/Twig/Tests/Extension/AbstractBootstrap3LayoutTest.php

```diff
@@ -2089,6 +2089,22 @@ public function testNumber()
         );
     }
 
+    public function testRenderNumberWithHtml5NumberType()
+    {
+        $form = $this->factory->createNamed('name', 'Symfony\Component\Form\Extension\Core\Type\NumberType', 1234.56, [
```

Comment:
In addition to `1234.56`, would it make sense to test what happens with `1,234.56` ? Thanks.

Reply:
As far as I know that's not something the client would send in this scenario. Or do you mean having a test that makes sure we get a transformation failure?

Reply:
@javiereguiluz I added such a test case to make sure that we provide a nice error message when you try to use the `grouping` option for HTML5 widgets. Good for you?

Reply:
Yes! Thank you.